src/test/java/net/openhft/chronicle/map/ExitHookTest.java

```diff
@@ -176,6 +178,32 @@ public void testSkipExitHook() throws IOException, InterruptedException {
         waitForShutdownConfirmation(shutdownActionConfirmationFile, USER_SHUTDOWN_HOOK_EXECUTED, process);
     }
 
+    @Test
+    public void testSerialization1() throws Exception {
+        File mapFile = folder.newFile();
+        ChronicleMap<Integer, Integer> expected = createMapBuilder()
+                .skipCloseOnExitHook(true)
+                .createPersistedTo(mapFile);
+        ChronicleMap<Integer, Integer> actual = createMapBuilder()
+                .createPersistedTo(mapFile);
+        Field skipCloseOnExitHook = VanillaChronicleHash.class.getDeclaredField("skipCloseOnExitHook");
```

Comment:
minor (I'll approve anyway) but if we are going to use reflection to get access to a field then it would perhaps be nice to extract method lines 189-190 so that a) that code is deduplicated and b) the reflection hackiness is in one place

Reply:
Done